nft-book-mint-and-list/iscn.js

```diff
@@ -84,7 +89,9 @@ export function validateISCNPrefix(input) {
 
 export async function getToken() {
   try {
-    const aminoSigner = await Secp256k1HdWallet.fromMnemonic(MNEMONIC, { prefix: 'like' });
+    const aminoSigner = MNEMONIC 
+      ? await Secp256k1HdWallet.fromMnemonic(MNEMONIC, { prefix: 'like' })
+      : await Secp256k1Wallet.fromKey(PRIVATE_KEY, { prefix: 'like' });
```

Comment:
I think `DirectSecp256k1Wallet.fromKey` accept prefix as string param, i.e. `await DirectSecp256k1Wallet.fromKey(PRIVATE_KEY, 'like');`
Please verify if thats the case